ROADMAP.md

```diff
@@ -0,0 +1,46 @@
+Roadmap
+=======
+
+These are the themes that we plan to work on for cert-manager. If you wish
+to discuss these topics you can find us in #cert-manager on Kubernetes Slack, or
+at our [community meetings](https://cert-manager.io/docs/contributing/#meetings).
+
+The roadmap items are categorized in to themes based on the larger goals we
+want to achieve with cert-manager.
+
+While this is a summary of the direction we want to go, we welcome all PRs,
+even if they don't fall under any of the roadmap items.
+
+* Beyond Ingress: improve experience of cert-manager for applications beyond just
+  ingress certificates
+  * Service Mesh Integration: Enable service meshes to issue mTLS certificates
+    with cert-manager, getting the integration with external issuers and the
+    audit capabilities of cert-manager in their mesh
+  * Istio agent certificates issued via cert-manager
+  * CSI driver: seamlessly deliver unique certs + keys to workloads. Review the
```

Comment:
`seamlessly deliver unique certs + keys to workloads`

I think it'd also be worth looking at ways we can make a more opinionated variant of the CSI driver that delivers identity certificates to pods transparently, without all the `volumeAttributes` used to specify what fields should be set on the certificate (i.e. delivering 'identity documents' to each pod as opposed to allowing pods to mount arbitrarily specified certificates)

Reply:
Yes, that's something that came up in a discussion the other day as it happens.